src/tests/jest.setup.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
```

Comment:
May be this file could be called `enzyme.setup.js` or `enzyme.config.js` because I see here you are configuring enzyme.

Reply:
I've renamed to `enzyme.config.js`